src/Stratis.Bitcoin.Tests/Utilities/DBreezeTest.cs

```diff
@@ -137,6 +137,28 @@ public void DeserializerWithNotSupportedClassThrowsException()
             });
         }
 
+        private class UnknownBitcoinSerialisable : IBitcoinSerializable
```

Comment:
It looks better if we consistently keep the American spelling of serializable I think, as that has less potential for confusion.

Reply:
I can't decide if I should please you or the code policy enforcement administration (from codereview-policy.md, l.43):
* **Typos and grammar**: make sure the code reads smoothly and is in British English.

Reply:
That is already not 100% achievable unless we substantially modify NBitcoin. But as this is just a test case there is no need to change it.

Reply:
UK spelling would make more sense for us, since we're a UK company ;-)